pkg/util/etcdutil/migrate.go

```diff
@@ -0,0 +1,175 @@
+package etcdutil
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/coreos/etcd/clientv3"
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/restclient"
+	"k8s.io/kubernetes/pkg/client/unversioned"
```

Comment:
As I understand it, the unversioned client *can* change within a release. So we could re-vendor kubernetes and potentially have to update code that uses the unversioned client. Whereas the generated clientset releases should not change (and when you're ready you can then switch to `release_1_5` for example, and handle all changes at once).

However, it doesn't look like you're using parts of the client that are likely to change much -- so this is not a big deal.